stanford-earth/SE3_BLT
# Slogan Setting

# READY FOR REVIEW

# Summary
- Changes the site name and slogan for each environment

# Needed By (Date)
- 2017.06.12

# Urgency
- Minor

# Steps to Test

1. Check out this branch on your local environment
2. run `drush csim local`
3. View website for new title and slogan

# Affected Projects or Products
- Will help https://github.com/stanford-earth/matson/pull/10/files

# Associated Issues and/or People
- EARTH-11
- https://github.com/stanford-earth/matson/pull/10/files
- @slowbot 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
